securitycenter/assets/assets_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package assets
+
+import (
+	"bytes"
+	"os"
+	"testing"
+	"time"
+)
+
+func setup(t *testing.T) string {
+	orgID := os.Getenv("GCLOUD_ORGANIZATION")
+	if orgID == "" {
+		t.Skip("GCLOUD_ORGANIZATION not set")
+	}
+	return orgID
+}
+
+func TestListAllAssets(t *testing.T) {
+	buf := new(bytes.Buffer)
+	orgID := setup(t)
+	got, err := listAllAssets(buf, orgID)
+	if err != nil {
+		t.Fatalf("listAllAssets(%s) failed: %v", orgID, err)
+	}
+	wanted := 59
```

Comment:
This should be used in the next line (`if got < want`). Totally minor and feel free to ignore, but this is usually just named `want`.